keras_cv/models/backbones/effnet_lite/effnet_lite_backbone.py

```diff
@@ -0,0 +1,494 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""EfficientNet Lite models for Keras.
+
+Reference:
+- [EfficientNet: Rethinking Model Scaling for Convolutional Neural Networks](
+        https://arxiv.org/abs/1905.11946) (ICML 2019)
+- [Based on the original EfficientNet Lite's]
+(https://github.com/tensorflow/tpu/tree/master/models/official/efficientnet/lite)
+"""
+
+import copy
+import math
+
+from tensorflow import keras
+from tensorflow.keras import backend
+from tensorflow.keras import layers
+
+from keras_cv.models import utils
+from keras_cv.models.backbones.backbone import Backbone
+from keras_cv.models.backbones.effnet_lite.effnet_lite_backbone_presets import (
```

Comment:
I think it should go with the full name, i.e. `efficientnet/efficientnet_lite_backbone.py` instead of `effnet_lite/effnet_lite_backbone.py`.

Reply:
@innat-asj , the problem I was facing when writing the full name was that it was crossing the 80 character limit and I don't know how to go around it. Can you suggest a solution ?

Reply:
Maybe (untested)

```python
from keras_cv.models.backbones.efficientnet_lite.\
	efficientnet_lite_backbone_presets import ( 
    	backbone_presets
	)
```

cc. @ianstenbit

Reply:
you can also at # noqa E501 to the end of the lines!

Reply:
Thanks! , will update it within a week !